scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -1624,56 +1678,96 @@
                          ; Need to pass in resources to task-metadata->pod for gpu count
                          :resources pool-specific-resources}})))))
 
+(defn create-kubernetes-jobs-launcher
+  "Create a function that will launch given jobs in a given compute cluster for
+   handling by the Kubernetes Scheduler."
+  [conn pool-name mesos-run-as-user]
+  (fn [[compute-cluster jobs]]
+    (let [cc-name (cc/compute-cluster-name compute-cluster) 
+          task-metadata-seq (jobs->kubernetes-task-metadata jobs pool-name mesos-run-as-user compute-cluster)
+          psuedo-matches [{:task-metadata-seq task-metadata-seq}]
+          task-txns (map (fn [job metadata] (job->task-txn job metadata compute-cluster)) jobs task-metadata-seq)]
+      (launch-tasks-for-cluster compute-cluster psuedo-matches task-txns pool-name conn nil false)
+      (log-structured/info "Launched jobs in Kubernetes compute cluster" {:pool pool-name
+                                                                          :compute-cluster cc-name
+                                                                          :number-launched-jobs jobs}))))
+
 (defn distribute-jobs-to-kubernetes
```

Comment:
Just realized that this distributes and launches. Maybe rename it to 'scheduler-jobs-on-kubernetes'?

Reply:
done